### ISSUE TITLE
Not playing Videos.

### ISSUE THREAD

**Issue by sebamena** [Krupen/AutoplayVideos#49]
When i tried to view more than 2 videos, the second video dont show and the videoplayer comes to blank.
I follow the examples, but the problem still appears.

When the activity is displayed, the items have a title, when the title is clicked, the cam opened and record the video and when finish, the video is displayed in the autoplayrecyclerview, but the video dont show automatic, and force to playvideo, but dont works.

When is only one video, all works fine, but if are two or more, i have problems to play videos correctly.

**Comment by sebamena** [comment]
After a lot of tests, when load the second video and scroll up and down the video appears, but i need to play the video automatic when the view of recyclerview is in the screen,

**Comment by sebamena** [comment]
I continue testing, and the problem is with the views, if the item view dont show in the screen totally, the video dont show.

I reduce the padding and margins, to show the item_video in the screen completly and the video it's showing!.
But what is my problem, and how fix this problem?

**Comment by Krupen** [comment]
Hi @sebamena ,
I believe you need to try calling `adapter.notifyDataSetChanged()` and then call following two methods to auto-play videos:
```
recyclerView.smoothScrollBy(0,1);
recyclerView.smoothScrollBy(0,-1);
```

Also for playing partially visible videos, I released an update(v0.2.0) in which you can set how much percentage the view needs to be visible in order for the video to start playing. In your case, you can set it as follows:
`recyclerView.setVisiblePercent(0);`

**Comment by Krupen** [comment]
I hope the issue is resolved. Please feel free to re-open the issue if not solved.
